ci/ansible/kiosk.yml

```diff
@@ -0,0 +1,32 @@
+- hosts: kiosks
+
+  vars:
+    - service_description: 'Herbert kiosk service'
+    - service_user: root
+    - service_file: dist/worker/main.js
```

Comment:
Should this be `"{{ appDir }}/scripts/kiosk.sh"` ?

Reply:
I did not mean to include this!